operators/cmd/manager/main.go

```diff
@@ -234,12 +240,27 @@ func execute() {
 		log.Error(err, "invalid roles specified")
 		os.Exit(1)
 	}
-	operatorNamespace := viper.GetString(OperatorNamespaceFlag)
-	log.Info("Setting up controller", "roles", roles)
+
+	// Setup a Kubernetes client without cache to request resources in the operator namespace and not in the
+	// managed namespaces until we upgrade the controller-runtime to be able to read in n namespaces with the
+	// cache enabled
+	operatorClient, err := k8s.NewClient(cfg)
+	if err != nil {
+		log.Error(err, "unable to set operator k8s client")
+		os.Exit(1)
+	}
+
+	// Setup operator info
+	operatorUUID, err := about.GetOperatorUUID(operatorClient, operatorNamespace)
+	operatorInfo := about.NewOperatorInfo(operatorUUID, operatorNamespace, cfg)
```

Comment:
I think we miss a unit test that focuses on L244-L255 (btw they can all be encapsulated in a single `about.GetOperatorInfo() call`?):

* Setup a fake client
* Run `about.GetOperatorInfo`
* Check that a configmap was created in the fake client (2nd test if the configmap already exists)
* Check the content of the info we retrieved

Reply:
oh right, I missed the point we are not using our usual k8s client here!
If that ends up adding more complexity e2e tests are maybe enough!